From cc50cdbcf3e8f065bd7798a92689f54578b4169f Mon Sep 17 00:00:00 2001
From: Richard Oliver <richard.oliver@raspberrypi.com>
Date: Wed, 24 Jul 2024 15:48:22 +0100
Subject: [PATCH 1251/1350] spi: rp2040-gpio-bridge: add missing MD5 dependency

rp2040-gpio-bridge relies on the md5 crypto driver. This dependency
cannot be determined automatically as rp2040-gpio-bridge does not
use any of md5's symbols directly.

Declare a soft 'pre' dependency on md5 to ensure that it is included and
loaded before rp2040-gpio-bridge.

Signed-off-by: Richard Oliver <richard.oliver@raspberrypi.com>
---
 drivers/spi/spi-rp2040-gpio-bridge.c | 1 +
 1 file changed, 1 insertion(+)

--- a/drivers/spi/spi-rp2040-gpio-bridge.c
+++ b/drivers/spi/spi-rp2040-gpio-bridge.c
@@ -1241,3 +1241,4 @@ module_i2c_driver(rp2040_gbdg_driver);
 MODULE_AUTHOR("Richard Oliver <richard.oliver@raspberrypi.com>");
 MODULE_DESCRIPTION("Raspberry Pi RP2040 GPIO Bridge");
 MODULE_LICENSE("GPL");
+MODULE_SOFTDEP("pre: md5");
